.github/workflows/sentry-ember.yml

```diff
@@ -0,0 +1,46 @@
+name: '@sentry/ember'
+
+on:
+  pull_request:
+    branches: [master]
+    paths:
+      - 'packages/ember/**'
+
+env:
+  lerna-package: '@sentry/ember'
+
+jobs:
+  test:
+    name: 'Test Ember Package'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Set up Volta
+        uses: volta-cli/action@v1
+
+      # See https://github.com/actions/cache/blob/master/examples.md#node---yarn for example
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v1
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install dependencies (yarn)
+        run: yarn install
+
+      - name: Lint
+        run: yarn run lint --scope=${{env.lerna-package}}
+
+      - name: Build dependent packages
+        run:  yarn build --ignore="@sentry/integrations" --ignore="@sentry/ember" --ignore="@sentry/react" --ignore="@sentry/apm" --ignore="@sentry/gatsby" --ignore="@sentry/node"
```

Comment:
I think @HazAT just converted a bunch to GHA, can we just leverage those and add ember under `test`?

We don't need to duplicate `lint` and `build` that way.

Reply:
I'll take a look, 👍

Reply:
Yeah took a look at the the GHA workflow and removed this, it's a lot simpler to just run the build for everything, it probably isn't worth dealing with managing individual workflows for a small speed increase.